app/src/main/java/org/yaaic/activity/AddServerActivity.java

```diff
@@ -333,9 +334,7 @@ private Server getServerFromView()
         String password = ((EditText) findViewById(R.id.password)).getText().toString().trim();
         String charset = ((Spinner) findViewById(R.id.charset)).getSelectedItem().toString();
         Boolean useSSL = ((CheckBox) findViewById(R.id.useSSL)).isChecked();
-
-        // not in use yet
-        //boolean autoConnect = ((CheckBox) findViewById(R.id.autoconnect)).isChecked();
+        Boolean autoConnect = ((CheckBox) findViewById(R.id.autoconnect)).isChecked();
```

Comment:
Instead of the Boolean object, use the atomic type boolean. Or do you need an object here?